api-schema/src/main/java/org/stellar/anchor/api/rpc/method/NotifyOffchainFundsReceivedRequest.java

```diff
@@ -26,5 +27,9 @@ public class NotifyOffchainFundsReceivedRequest extends RpcMethodParamsRequest {
   private AmountRequest amountOut;
 
   @SerializedName("amount_fee")
+  @Deprecated
   private AmountRequest amountFee;
+
+  @SerializedName("fee_details")
+  private RateFee feeDetails;
```

Comment:
`RateFee` used for `feeDetails` seems a little counter intuitive. Is `RateFee` used somewhere else? Or can we use a better name?

Reply:
Yes, RateFee is used in SEP-38 in SEP-38 quote. The SEP was designed to match the SEP-38 quote fee structure exactly.
I don't mind renaming it, maybe 
`RateFee` -> `FeeDetails`
`RateFeeDetail` -> `FeeDescrption` 
WDYT? (cc @philipliu on this)

Reply:
That sounds good 👍